apps/rabbitmq/4.0.4-management-alpine/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   rabbitmq:
-    image: rabbitmq:4.0.4-management-alpine
+    image: rabbitmq:4.0.5-management-alpine
     hostname: rabbitmq
     container_name: ${CONTAINER_NAME}
     restart: always
```

Comment:
There is an issue with the Dockerfile at line 3. The version of RabbitMQ was updated from "4.0.4" to "4.0.5", but there are no changes specified within the `Dockerfile` that correspond to this minor update. If you intend to upgrade to RabbitMQ version 4.0.5, please ensure additional configuration adjustments or environment variables specific to this version are included in your `docker-compose.yml`.